cmd/gofmtmd/main.go

```diff
@@ -19,10 +19,19 @@ var rootCmd = &cobra.Command{
 	Use:     "gofmtmd",
 	Version: "0.1.1",
 	Short:   "This CLI formats Go Code in Markdown.",
-	Args:    cobra.MinimumNArgs(1),
+	Args:    cobra.MinimumNArgs(0),
```

Comment:
If you want to receive less than one argument, I think we should use ```MaximumNArgs```.

```diff
- Args:    cobra.MinimumNArgs(0),
+ Args:    cobra.MaximumNArgs(1),
```

Reply:
Thank you!
I didn't know `MaximumNArgs`